draft-dekater-scion-pki.md

```diff
@@ -1303,8 +1335,42 @@ The steps required to create a new AS certificate are the following:
+
+Another possible attack is when a core AS stops propagating PCBs, thus frustrating the discovery of new paths. In this case, downstream ASes will notice that PCBs are no longer being propagated, but all previously discovered (and still valid) paths are still usable for data-plane forwarding until they expire.
+
+Perhaps a more stealthy kill switch would be to shut down control services in victim ASes. An adversarial entity controlling an ISD (e.g., a government) might compel core and non-core ASes to stop replying to path requests. Alternatively, the compelled ASes might return only a subset of all available paths. If this attack were used in conjunction with blackholing, where traffic is redirected to a non-existent resource, senders in the ISD would have difficulty getting traffic out of the ISD. In SCION, however, existing paths can continue to be used in the data plane as long as the traversed ASes allow the forwarding.
+
+
+### Remote ISD/AS Kill Switch
+
+SCION ISDs independently manage their own cryptographic keys and namespace. This prevents a remote attacker who is outside the victim's ISD from causing a kill switch in the victim ISD. Without access to the private keys that form the trust root in the victim ISD, the remote attacker is limited to data-plane attacks. Even if private keys became available to a remote attacker, they would need access to an AS inside the remote ISD to inject faulty information.
+
+
+### Recovery from Kill Switches
+
+In the event of a key compromise of a non-core AS, the impacted AS needs to obtain a new certificate from the core. This process will vary depending on internal issuance protocols. If any of the root keys or voting keys contained in the TRC are compromised, the TRC must be updated as described in [](#update). A trust reset is only required in the case of a catastrophic compromise of multiple voting keys at the same time.
+
+If the core AS has not been compromised, but is instead acting maliciously (e.g., by not propagating PCBs downstream or by falsifying responses to paths- or certificates requests), one way to recover is for downstream ASes to self-organize and form a new ISD. By now operating autonomously, the new ISD can begin path discovery and traffic forwarding. SCION, unlike BGP, has no notion of routing convergence. Instead, the flooding of PCBs disseminates topology information. This means that in the worst case, if all paths must be re-created, fresh paths are established after a single flood has reached all ASes.
```

Comment:
Again, I think we should focus on the CP-PKI here. We can replace the examples for "acting maliciously" to, "e.g. by not issuing or renewing certificates for certain ASes". 

Reading this though, it's not clear that a core AS acting in such a way can even be considered malicious. It's the job of the core AS to enforce an ISDs policy. So the message is more "If your not happy with an ISDs policy, then don't join this ISD". This seems rather self-evident, to be honest, and doesn't really fit in these security considerations. Maybe we can leave this paragraph away.

Reply:
@matzf: good consideration.  I have added a note about it to the text.

Reply:
> Reading this though, it's not clear that a core AS acting in such a way can even be considered malicious. 
I agree, maybe we should remove that paragraph.

Reply:
Removed in https://github.com/scionassociation/scion-cppki_I-D/pull/25/commits/6cb1402685fabf6c33241cfb70c065eb203379b2